ShortenLink/Scenes/LinksPopOver/LinksPopOverViewController.swift

```diff
@@ -65,5 +89,68 @@ extension LinksPopOverViewController {
         settingButton.bezelStyle = .shadowlessSquare
         settingButton.isBordered = false
         settingButton.imagePosition = .imageOnly
+
+        let column = NSTableColumn(identifier: NSUserInterfaceItemIdentifier(rawValue: "column"))
```

Comment:
Let's make a static var for the Identifier string `column`

Reply:
I don't think it is necessary to create a static var for this case. I'm not using the custom view like the `ShortenLinkCell`